src/nep.h

```diff
@@ -43,9 +43,9 @@ class NEP3
     int dim = 0;
     int num_neurons1 = 0;
     int num_para = 0;
-    const double* w0;
-    const double* b0;
-    const double* w1;
+    const double* w0[100];
+    const double* b0[100];
+    const double* w1[100];
```

Comment:
So there can be no more than 100 different atom types, and thus a maximum of a hundred different networks?

Reply:
yes, 100 is just my estimate for the maximum number of elements in the potential :-)

Reply:
Sounds reasonable, if someone tries to run a simulation with more than 100 species they will have other problems :rofl: